libsrc/_DEVELOPMENT/target/clib_variables.inc

```diff
@@ -38,27 +38,55 @@ IF __clib_malloc_heap_size = -1
       
    ld hl,-1                    ; ensure sp == 0 is accepted
    add hl,sp                   ; hl = top of the stack
-   
+
+IF __CPU_INTEL__ || __CPU_GBZ80__
```

Comment:
We could make this cleaner by always using bc as __BSS_END_tail the the CPU variation is just on the subtraction

Reply:
I am not sure that the contents of `bc` aren't otherwise needed.
But there's definitely a better cleaner way to do it, which I didn't do (trying to not affect other code paths).